### PR TITLE
Account for some html layout changes

### PR DESCRIPTION
**Pull request by hc5duke**
https://app.honeybadger.io/projects/50956/faults/78788058

play store html layout has changed and some of the fields are no longer in the same place. I ignored some of the field we are not using, and tried to fix some of the other fields. `version` is very wonky and I'm not even sure if this would work 100% of the time, but I'm not sure it's a field we absolutely need either.

### CODE REVIEW

File: spec/market_bot/play/app_spec.rb [hunk 1]
```diff
@@ -3,18 +3,18 @@
 describe MarketBot::Play::App do
   shared_context('parsing an app') do
     it 'should parse the category attribute' do
-      expect(@parsed[:category]).to eq('Arcade').or eq('Weather').or eq('Simulation')
+      expect(@parsed[:category]).to eq('Arcade').or eq('Weather').or eq('Simulation').or eq('ARCADE').or eq('WEATHER').or eq('SIMULATION')
     end
 
-    it 'should parse the category_url attribute' do
+    xit 'should parse the category_url attribute' do
```
Comment (hc5duke) [inline spec/market_bot/play/app_spec.rb:9]: fields we don't need and is erroring out are ignored with `xit`

**Review by amirmanji — APPROVED** [review]
In terms of data we get from this, I think the only must-haves are:
- store_id / bundle_id
- app name
- app icon

The rest (genre, developer, etc) is just nice-to-have.

**Comment by amirmanji** [comment]
I think the branch we use in tracker is `ignore_more_from_developer`: https://github.com/tenjin/tracker/blob/8552e7c34f7e9662deb7f7b56219bc0cf6f1a576/Gemfile#L132-L134

**Comment by amirmanji** [comment]
So I think we'll either need to merge this PR into that branch or have tracker point at a different branch (either `master` or a branch specific to this PR).

File: lib/market_bot/play/app.rb [hunk 2]
```diff
@@ -75,21 +75,59 @@ def self.parse(html, _opts = {})
...
+            result[:developer]     = data.dig('author', 'name')
+            result[:developer_url] = developer_div.css('a').attr('href').value
+            result[:developer_id]  = result[:developer_url].split('?id=').last.strip
+
+            result[:rating] = data.dig('aggregateRating', 'ratingValue')
+            result[:votes]  = data.dig('aggregateRating', 'ratingCount').to_i
+
+            result[:cover_image_url] = data['image']
+
+            result[:updated] ||= text = doc.search("meta[itemprop='description']")[0].parent.children.last.children.last.children.last.text
```
Comment (makototaguchi1110) [inline lib/market_bot/play/app.rb:100]: @hc5duke `result[:updated]` becomes some weird value: `Stylized` and it throws error [here](https://github.com/tenjin/tracker/blob/master/lib/tracker/app_store.rb#L113). Do you know if we can fix it? Or we could just assign `nil` for now.

**Pull request merged**